beacon-chain/sync/validate_blob.go

```diff
@@ -133,22 +134,26 @@ func (s *Service) validateBlob(ctx context.Context, pid peer.ID, msg *pubsub.Mes
 
 	blobSidecarArrivalGossipSummary.Observe(float64(sinceSlotStartTime.Milliseconds()))
 
-	msg.ValidatorData = sBlob
+	vblob, err := verification.BlobSidecarNoop(blob)
+	if err != nil {
+		return pubsub.ValidationReject, errors.New("TODO: undefined verification error")
```

Comment:
Is this going to be handled in another PR?

Reply:
yes, we'll get rid of the "fake" verification code in the next PR. Currently this is unreachable as `BlobSidecarNoop` never returns an error.